### PR TITLE
Fix removing participants

### PR DESCRIPTION
**Pull request by ThiefMaster**
Now it works for both existing and new participants

### CODE REVIEW

File: newdle/client/src/reducers/creation.js [hunk 1]
```diff
@@ -84,7 +84,12 @@ export default combineReducers({
       case ADD_PARTICIPANTS:
         return [...state, ...action.participants];
       case REMOVE_PARTICIPANT:
-        return state.filter(p => p.id !== action.participant.id);
+        if (action.participant.id) {
+          return state.filter(p => !p.id || p.id !== action.participant.id);
```
Comment (plourenco) [inline newdle/client/src/reducers/creation.js:88]: Why not just?
```suggestion
          return state.filter(p => p.id !== action.participant.id);
```

Comment (ThiefMaster) [reply newdle/client/src/reducers/creation.js:88]: newly added participants from user search don't have one

Comment (plourenco) [reply newdle/client/src/reducers/creation.js:88]: But wouldn't in that case `p.id` still be different from `action.participant.id`? As we never have a `undefined` `action.participant.id`.

Comment (ThiefMaster) [reply newdle/client/src/reducers/creation.js:88]: It would be, and thus remove all those participants

Comment (ThiefMaster) [reply newdle/client/src/reducers/creation.js]: ok nevermind, i was somehow thinking the other way round. you were right that the first half of that check isn't needed.

**Pull request merged**